source/install/config-tls-mattermost.rst

```diff
@@ -42,3 +42,18 @@ The easiest option is to set up TLS on the Mattermost Server, but if you expect
 
 .. note::
    Password-protected certificates are not supported.
+   
+**Configure TLS on NGINX(proxy)**:
```

Comment:
```suggestion
**Configure TLS on NGINX (proxy)**
```

Reply:
Are the steps below a procedure or are they guidelines? If they're guidelines, one option is to replace the numbers with bullet points instead. :)